core-tests/src/test/scala/swave/core/impl/stages/SimpleOpSpec.scala

```diff
@@ -280,6 +280,13 @@ final class SimpleOpSpec extends SyncPipeSpec with Inspectors {
     }
   }
 
+  "Sliding" in check {
```

Comment:
Nice! This is the first time someone uses this new fancy test generation kit... :)